### PR TITLE
Disable package parametrizations until they are fixed

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

Avoiding red CI.

### Does your PR introduce any breaking changes? If yes, please list them.

None

cc @carmocca @akihironitta @borda

### CODE REVIEW

**Review by otaj — APPROVED** [review]

**Comment by Borda** [comment]
will re-enabled in #15105

File: .github/workflows/ci-app-examples.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macOS-11, windows-2022]
-        pkg-name: ["LAI", "app"]  # the "LAI" is for installing monolithic package
+        pkg-name: ["app"]  # TODO: package parametrization
```
Comment (Borda) [inline .github/workflows/ci-app-examples.yml:31]:
```suggestion
        pkg-name: ["app"]  # TODO: add "LAI" with mirror package # the "LAI" is for installing monolithic package
```

File: .github/workflows/ci-app-tests.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macOS-11, windows-2022]
-        pkg-name: ["LAI", "app"]  # the "LAI" is for installing monolithic package
+        pkg-name: ["app"]  # TODO: package parametrization
```
Comment (Borda) [inline .github/workflows/ci-app-tests.yml:35]:
```suggestion
        pkg-name: ["app"]  # TODO: add "LAI" with mirror package # the "LAI" is for installing monolithic package
```

File: .github/workflows/ci-lite-test-full.yml [hunk 3]
```diff
@@ -34,7 +34,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, windows-2022, macOS-11]
-        pkg-name: ["LAI", "lite"]  # the "LAI" is for installing monolithic package
+        pkg-name: ["lite"]  # TODO: package parametrization
```
Comment (Borda) [inline .github/workflows/ci-lite-test-full.yml:37]:
```suggestion
        pkg-name: ["lite"]  # TODO: add "LAI" with mirror package # the "LAI" is for installing monolithic package
```

File: .github/workflows/ci-pytorch-test-full.yml [hunk 4]
```diff
@@ -37,7 +37,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, windows-2022, macOS-11]
-        pkg-name: ["LAI", "pytorch"]  # the "LAI" is for installing monolithic package
+        pkg-name: ["pytorch"]  # TODO: package parametrization
```
Comment (Borda) [inline .github/workflows/ci-pytorch-test-full.yml:40]:
```suggestion
        pkg-name: ["pytorch"]  # TODO: add "LAI" with mirror package # the "LAI" is for installing monolithic package
```

File: .github/workflows/code-checks.yml [hunk 5]
```diff
@@ -13,6 +13,7 @@ concurrency:
 jobs:
   mypy:
     runs-on: ubuntu-20.04
+    # TODO: package parametrization
```
Comment (Borda) [inline .github/workflows/code-checks.yml:16]:
```suggestion
```
Why this shall be parametrized?

**Review by Borda — APPROVED** [review]

File: .github/workflows/code-checks.yml (hunk 5 above)
Comment (carmocca) [reply .github/workflows/code-checks.yml:16]: I can imagine that there are mypy issues after rewriting the imports

**Review by akihironitta — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

File: .github/workflows/ci-app-examples.yml [hunk 6]
```diff
@@ -32,7 +32,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macOS-11, windows-2022]
-        pkg-name: ["LAI", "app"]  # the "LAI" is for installing monolithic package
+        pkg-name: ["app"]  # TODO: add "LAI" # "LAI" installs the monolithic package
```
Comment (Borda) [inline .github/workflows/ci-app-examples.yml:35]:
```suggestion
        pkg-name: ["app", "LAI"]  # "LAI" installs the monolithic package
```
this is already working

File: .github/workflows/ci-pytorch-test-full.yml [hunk 7]
```diff
@@ -115,9 +115,8 @@ jobs:
         pip list
 
     - name: DocTests PL
-      # todo: run docstest laos on mono-package
```
Comment (Borda) [inline .github/workflows/ci-pytorch-test-full.yml:118]: Why is this removed?

File: .github/workflows/ci-app-examples.yml (hunk 6 above)
Comment (carmocca) [reply .github/workflows/ci-app-examples.yml:35]: Let's see if it's true

Comment (carmocca) [reply .github/workflows/ci-app-examples.yml]: Still failing: 

https://github.com/Lightning-AI/lightning/actions/runs/3313663726/jobs/5471949250

File: .github/workflows/ci-app-examples.yml [hunk 8]
```diff
@@ -32,7 +32,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macOS-11, windows-2022]
-        pkg-name: ["LAI", "app"]  # the "LAI" is for installing monolithic package
+        pkg-name: ["app", "LAI"]  # "LAI" installs the monolithic package
```
Comment (carmocca) [inline .github/workflows/ci-app-examples.yml:35]:
```suggestion
        pkg-name: ["app"]  # TODO: add "LAI" # "LAI" installs the monolithic package
```

**Comment by otaj** [comment]
It seems quite a few tests are failing with installation step with "no matching distribution found"

**Comment by carmocca** [comment]
I'm seeing the error on other PRs. Looks like a PyPI issue

**Comment by carmocca** [comment]
https://status.python.org/

**Comment by otaj** [comment]
> I'm seeing the error on other PRs. Looks like a PyPI issue

Yep: https://status.python.org/

**Comment by tchaton** [comment]
@carmocca @Borda Any ETA for this one to be merged? Currently blocking some PRs for tomorrow's workshop.

**Comment by carmocca** [comment]
There's a regression that is causing jobs to hang after pytest finishes. The global timeout gets triggered ~30 min later. The cause is unknown so far.

**Comment by carmocca** [comment]
It's only impacting PL tests as far as I can tell, so if it's blocking apps, then we can ask to force merge this one and debug that separately

**Comment by Borda** [comment]
> @carmocca @Borda Any ETA for this one to be merged? Currently blocking some PRs for tomorrow's workshop.

This is not blocking anything, just shot not required test failing as they are expected to fail...
